src/president/hand.js

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
I wondered if the hand module is relevant because it seems to be the same as deck.

Reply:
I could merge the two in one module cards..

Reply:
Deal!